optimum/onnxruntime/quantization.py

```diff
@@ -356,62 +356,45 @@ def quantize(
                 )
 
         quantizer_factory = QDQQuantizer if use_qdq else ONNXQuantizer
+        # TODO: maybe this logic can be moved to a method in the configuration class (get_ort_quantizer_kwargs())
```

Comment:
But the config should not be aware of the ORTQuantizer class right?

Reply:
yes, the quant config already contains everything and can infer which quantizer will use its kwargs (from format and is_static https://github.com/huggingface/optimum/blob/f3008651c6f674d4b89de66a2d21fc5e7cafaf84/optimum/onnxruntime/quantization.py#L309)